addons/binding/org.openhab.binding.homie/src/main/java/org/openhab/binding/homie/internal/conventionv200/NodePropertiesListAnnouncementParser.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.homie.internal.conventionv200;
+
+import static org.openhab.binding.homie.internal.conventionv200.HomieConventions.ID_PATTERN;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Parser for property announcement messages (e.g. the message sent to homie/686f6d6965/temperature/$properties)
+ *
+ * @author Michael Kolb - Initial Contribution
+ *
+ */
+public class NodePropertiesListAnnouncementParser {
+
+    protected final static String MATCHGROUP_PROPERTYNAME_NAME = "propname";
```

Comment:
It is the name of the matchgroup that matches a propertyname. So it is correct as it is.